martin/src/file_config.rs

```diff
@@ -40,6 +39,22 @@ pub enum FileConfigEnum {
     Config(FileConfig),
 }
 
+impl FileConfigEnum {
+    pub fn extract_file_config(&mut self) -> FileConfig {
+        match self {
+            FileConfigEnum::Path(path) => FileConfig {
+                paths: Some(OneOrMany::One(mem::take(path))),
```

Comment:
nit: This looked cleaner when `One` and `Many` were imported directly

Reply:
fixed.  This one is kinda tricky because often when you see `One`, it might not give enough context - so i have seen a lot of things like `io::Result` when things are not extra clear.